packages/salesforcedx-vscode-core/src/conflict/index.ts

```diff
@@ -77,3 +78,9 @@ function openResource(node: ConflictNode) {
     window.showTextDocument(local).then(() => { });
   }
 }
+
+export function setupFilePropertyStorage(
+  extensionContext: ExtensionContext
+) {
+  PersistentStorageService.initialize(extensionContext);
+}
```

Comment:
do we need to have a wrapper around the initialize() here? could we call initialize() directly when needed?

Reply:
Updated it to do that in the latest commit